timetable.md

```diff
@@ -91,3 +91,17 @@
 ## 9.2 – Thu, December 15, 2022
 
 - The Challenge, step two, presentations
+
+## 10.1 – Thu, January 12, 2023
+
+- **10** [Organizational remarks](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/00_organization/organizational_remarks_week10_slides.md)
+- **25** Markup Languages: [slides](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/04_documentation/markup_languages_slides.md)
+- **20** Pandoc: [slides](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/04_documentation/pandoc_slides.md), [demo](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/04_documentation/pandoc_demo.md)
+- **35** Documentation Website Generators: [slides](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/04_documentation/website_generators_slides.md), [demo](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/04_documentation/website_generators_demo.md)
+
+## 10.2 – Thu, January 12, 2023
+
+- **20** [Floss Licenses](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/06_miscellaneous/floss_licenses_slides.md)
+- **20** [Versioning](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/06_miscellaneous/versioning_slides.md)
+- **20** [Repository Layouts](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/06_miscellaneous/repository_layouts_slides.md)
+- **20** [DOI, Zenodo, DaRUS](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/06_miscellaneous/doi_zenodo_darus_slides.md)
```

Comment:
Adds up to 80 mins, just in case you are aiming for a full 90 mins schedule each time here.

Reply:
No, goal is to be as realistic as possible. Let's see :D